src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/ItemGroupLoggingHelper.cs

```diff
@@ -56,7 +53,8 @@ internal static string GetParameterText(string prefix, string parameterName, ILi
                     sb.Append("\n    ");
                 }
 
-                sb.Append(parameterName + "=");
+                sb.Append(parameterName);
+                sb.Append('=');
```

Comment:
At first, I thought this would allocate a new string for the `=` each time, but I see now it's a character, so that shouldn't be a problem.

Reply:
String literals that are compile-time constants are stored in the Strings metadata heap and don't need to be allocated.